src/Mod/Fem/femsolver/ratel/tasks.py

```diff
@@ -80,31 +81,73 @@ def run(self):
             self.fail()
         _inputFileName = os.path.splitext(os.path.basename(path))[0]
         
+        
 
 class Solve(run.Solve):
 
     def run(self):
-        # self.pushStatus("Executing solver...\n")
-
-        # # get solver binary
-        # self.pushStatus("Get solver binary...\n")
-        # binary = settings.get_binary("Ratel")
-        # if binary is None:
-        #     self.pushStatus("Error: The Ratel binary has not been found!")
-        #     self.fail()
-        #     return
-
-        # # run solver
-        # self._process = subprocess.Popen(
-        #     [binary, "-i", _inputFileName],
-        #     cwd=self.directory,
-        #     stdout=subprocess.PIPE,
-        #     stderr=subprocess.PIPE
-        # )
-        # self.signalAbort.add(self._process.terminate)
+        self.pushStatus("Executing solver...\n")
+        
+        ratel_path, errors = ratel_util.get_location("ratel")
+        lib_ceed_path, errors = ratel_util.get_location("libCEED")
+        petsc_path, errors = ratel_util.get_location("petsc")
```

Comment:
This is ignoring errors, but also, we don't need these paths to be set to run the executable.